SECURITY-RELEASE-PROCESS.md

```diff
@@ -0,0 +1,193 @@
+# Security Release Process
+
+The CoreDNS project has adopted this security disclosures and response policy
+to ensure responsible handling of critical issues.
+
+
+## Product Security Team (PST)
+
+Security vulnerabilities should be handled quickly and sometimes privately. 
+The primary goal of this process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal communication and external disclosure.
+
+The initial Product Security Team will consist of the set of maintainers that volunteered.
+The list of members is identified in this [document](https://github.com/coredns/coredns/OWNERS) under the string `product-security-team` and is synchronized with the receivers of the members of security@coredns.io mailing list.
+
+### mailing lists
+
+* security@coredns.io : for any security concerns. Received by Product Security Team members, and used by this Team to discuss security issues and fixes.
+* distributors-announce@coredns.io: for early private information on Security patch releases. see below how CoreDNS distributors can apply for this list.
```

Comment:
the @coredns.io is a bit adhoc at the moment and requires me manually adding people (or paying $$ for a actual group), this should probably be a closed group on groups.google.com or something hosted by the cnfc we people can subscribe.

Reply:
possibly aliased from distributors-announce@coredns.io

Reply:
Do you mean we need to ask creation of the dl for the 2 lists or only for distributor.
Something like:
- coredns-distributor-announce@googlegroup.com (to warn the distributors of early release)
- coredns-security-team@googlegroup.com (to discuss about security patch, only the PST team members)

and keep security@coredns.io for external request on 'security' ?